integration/setup/teardown.go

```diff
@@ -21,12 +22,13 @@ func teardown(ctx context.Context, config Config) error {
 
 	{
 		releases := []string{
-			fmt.Sprintf("%s-aws-operator", config.Host.TargetNamespace()),
-			fmt.Sprintf("%s-cert-operator", config.Host.TargetNamespace()),
-			fmt.Sprintf("%s-node-operator", config.Host.TargetNamespace()),
```

Comment:
`Release.EnsureDeleted` prefixes the given names already with the target namespace. This here was broken.